server/plugins/renderer/renderer.module.js

```diff
@@ -458,24 +458,33 @@ internals.getHeaders = function (request, options, config) {
  * @type {Object}
  */
 internals.themeAssembler = {
-    getTemplates: (path, processor, callback) => {
-        TemplateAssembler.assemble(internals.getThemeTemplatesPath(), path, (err, templates) => {
-            if (templates[path]) {
-                // Check if the string includes frontmatter configuration and remove it
-                var match = templates[path].match(/---\r?\n[\S\s]*\r?\n---\r?\n([\S\s]*)$/);
-
-                if (_.isObject(match) && match[1]) {
-                    templates[path] = match[1];
+    getTemplates: (path, processor) => {
+        return new Promise((resolve, reject) => {
```

Comment:
This doesn't appear to be used in the below promise chain, why was it converted?